suites/squid/rados/tier-3_rados_cidr_blocklisting.yaml

```diff
@@ -0,0 +1,174 @@
+              args:          # arguments to ceph orch
+                - ceph
+                - fs
+                - volume
+                - create
+                - cephfs
+          - config:
+              command: apply
+              service: rgw
+              pos_args:
+                - rgw.1
+              args:
+                placement:
+                  label: rgw
+          - config:
+              command: apply
+              service: mds
+              base_cmd_args:          # arguments to ceph orch
+                verbose: true
+              pos_args:
+                - cephfs              # name of the filesystem
+              args:
+                placement:
+                  nodes:
+                    - node2
+                    - node6
+                  limit: 2            # no of daemons
+                  sep: " "            # separator to be used for placements
+      destroy-cluster: false
+      abort-on-fail: true
+
+
+  - test:
+      name: Configure clients
+      desc: Configures multiple client nodes on cluster
+      module: test_parallel.py
+      parallel:
+        - test:
+            name: Configure client 1
+            desc: Configures client.1 admin node on cluster
+            module: test_client.py
+            polarion-id:
+            config:
+              command: add
+              id: client.1                      # client Id (<type>.<Id>)
+              nodes:
+                  - node8:
+                      release: 8
+                  - node7:
+                      release: 8
+              install_packages:
+                - ceph-common
+              copy_admin_keyring: true          # Copy admin keyring to node
+              caps:                             # authorize client capabilities
+                mon: "allow *"
+                osd: "allow *"
+                mds: "allow *"
+                mgr: "allow *"
+        - test:
+            name: Configure client 2
+            desc: Configures client.2 admin node on cluster
+            module: test_client.py
+            polarion-id:
+            config:
+              command: add
+              id: client.2                     # client Id (<type>.<Id>)
+              nodes:
+                - node9:
+                    release: 8
+              install_packages:
+                - ceph-common
+              copy_admin_keyring: true          # Copy admin keyring to node
+              caps: # authorize client capabilities
+                mon: "allow *"
+                osd: "allow *"
+                mds: "allow *"
+                mgr: "allow *"
+        - test:
+            name: Configure client 3
+            desc: Configures client admin node on cluster
+            module: test_client.py
+            polarion-id:
+            config:
+              command: add
+              id: client.3                      # client Id (<type>.<Id>)
+              node: node10                       # client node
```

Comment:
RHCS 8 Clients would be installed on node 10, so test coverage is not lost.

Let's change the client deployment versions for node 7, 8, 9 above.

Reply:
Changed the node-7 and node-8 to 6,node-9 to 7 and node-10 nodes key does not exist so this client will pick the nightly RHCS-8 package.